spec/models/ring_spec.rb

```diff
@@ -4,14 +4,16 @@
   describe "#link" do
     it "adds a new redirection while preserving the ring" do
       new_redirection = build(:redirection)
```

Comment:
Why `build` here instead of `create`? Is it to do with callbacks for integrating into the ring?